src/components/InstanceMonitoring/InstanceMonitoringMetric.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const InstanceMonitoringMetric = props => (
```

Comment:
Nitpick - Do we want to use [object destructuring](https://davidwalsh.name/destructuring-function-arguments) in `props`, like `({ amount, unit, desc })` to avoid repeating `props.`?